File: 9225/timso.cpp
#include <bits/stdc++.h>
#define ll long long
using namespace std;
ll a, x, res;
int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL); cout.tie(NULL);
    freopen("timso.inp", "r", stdin);
    freopen("timso.out", "w", stdout);
    cin >> a >> x;
    for(int i = 1; i <= 40; i++)
    {
        if(pow(a, i)<=x)
        {
            res = i;
        }
    }
    cout << res;
    return 0;
}

File: timso.py
import sys
import math
#sys.stdin = open('timso.inp', 'r') 
#sys.stdout = open('timso.out', 'w')
a=int(input())
x=int(input())
for i in range(40):
    if(pow(a,i)<=x):
        print(i)
        #sys.exit()
#sys.close()